hooks/post/30-upgrade.rb

```diff
@@ -175,40 +63,7 @@ def fail_and_exit(message)
 
 if app_value(:upgrade)
   if [0, 2].include?(@kafo.exit_code)
-    upgrade_step :restart_services, :run_always => true
-
-    if Kafo::Helpers.module_enabled?(@kafo, 'katello')
-      upgrade_step :db_seed, :run_always => true
-
-      upgrade_step :correct_repositories, :long_running => true, :run_always => true
-      upgrade_step :correct_puppet_environments, :long_running => true, :run_always => true
-      upgrade_step :clean_backend_objects, :long_running => true, :run_always => true
-
-      upgrade_step :import_package_groups, :long_running => true
-      upgrade_step :import_rpms, :long_running => true
-      upgrade_step :import_distributions, :long_running => true
-      upgrade_step :import_puppet_modules, :long_running => true
-      upgrade_step :import_subscriptions, :long_running => true
-      upgrade_step :import_product_content, :long_running => true
-      upgrade_step :add_export_distributor, :long_running => true
-      upgrade_step :remove_docker_v1_content, :long_running => true
-      upgrade_step :update_puppet_repository_distributors, :long_running => true
-      upgrade_step :update_subscription_facet_backend_data, :long_running => true
-      upgrade_step :remove_event_queue
-      upgrade_step :set_virt_who_on_pools, :long_running => true
-      upgrade_step :remove_unused_products, :long_running => true
-      upgrade_step :create_host_subscription_associations, :long_running => true
-      upgrade_step :reindex_docker_tags, :long_running => true
-      upgrade_step :republish_file_repos, :long_running => true
-      upgrade_step :import_backend_consumer_attributes, :long_running => true
-      upgrade_step :remove_registration_tasks
-      upgrade_step :ensure_ks_repos_are_bootable, :long_running => true
-      upgrade_step :set_upstream_pool_id, :long_running => true
-    end
-
-    if [0, 2].include? @kafo.exit_code
-      Kafo::Helpers.log_and_say :info, 'Upgrade completed!'
```

Comment:
Sorry if I was unclear, but I meant that the `if` was redundant. The `Upgrade completed!` can still be useful though I don't mind a silent installer :)

Reply:
added it back below :)